packages/azure-services/src/storage/cosmos-container-client.spec.ts

```diff
@@ -32,7 +36,24 @@ beforeEach(() => {
     cosmosClientWrapperMock = Mock.ofType<CosmosClientWrapper>();
     operationCallbackMock = Mock.ofType<OperationCallback>();
     loggerMock = Mock.ofType(MockableLogger);
-    cosmosContainerClient = new CosmosContainerClient(cosmosClientWrapperMock.object, dbName, collectionName, loggerMock.object);
+    systemUtilsMock = Mock.ofType<typeof System>();
+    cosmosContainerClient = new CosmosContainerClient(
+        cosmosClientWrapperMock.object,
+        dbName,
+        collectionName,
+        loggerMock.object,
+        systemUtilsMock.object,
+    );
+    MockDate.set(startTime);
+    systemUtilsMock
+        .setup(s => s.wait(It.isAny()))
```

Comment:
now that you are mocking wait function, you can validate the argument passed to wait. (intervalMilliseconds)